dmd_G1Integral.m

```diff
@@ -0,0 +1,24 @@
+function G1Int=dmd_G1Integral(N,x,b)
```

Comment:
I would suggest implementing this short function into a subfunction of distanceDiracGaussian.m instead of in a separate file. This would keep it more readable.